Add blog-post about the year 2021 (#369)
packit/packit.dev: content/posts/2021-in-numbers.md

---
title: "2021 in Numbers"
date: 2022-01-04
---

# 2021 for Packit in numbers

Let's take a look on the last year through some numbers.

> We would like to should you some interesting statistics and charts

Reviewer: ```suggestion
We would like to show you some interesting statistics and charts
```

that can describe the work of Packit during the year 2021.

## Github Application

> By now, we have `169` installations of our GitHub application and the `41` is from the last year.

Reviewer: I'd say the 'by now' implies present perfect so I'd use either:

- By now, we've had `169` ...
- As of now, we have `169` ...

> By now, we have `169` installations of our GitHub application and the `41` is from the last year.

Reviewer: ```suggestion
By now, we have `169` installations of our GitHub application and `41` of them is from the last year.
```

> Looking at the monthly numbers bellow, it looks like we are getting back to shape.

Reviewer 2: ```suggestion
Looking at the monthly numbers below, it looks like we are getting back to shape.
```

![GitHub installations in 2021](/images/2021-in-numbers-github-installations.png)

## Builds

> Comparing to the previous year where we made `28 430` Copr builds for our users,

Reviewer 2: ```suggestion
Compared to the previous year when we made `28 430` Copr builds for our users,
```

we made `4.6` times more this year: `133 222` Copr builds.
For those who remember the start of our project,
we had a goal of `5` thousand for the [FLOCK](https://flocktofedora.org/) 2019.
We are now two digits ahead!
And if you are wondering how active is our user on Copr,

> we've created `2/3` of all the new Copr project during the year.

Reviewer: ```suggestion
we've created `2/3` of all the new Copr projects during the year.
```

To made this happen, we've created `36 133` source RPM files in the last year.

![Copr builds in 2021](/images/2021-in-numbers-copr-builds.png)

## Test runs

Sadly, we started saving the submit time of the test runs in June
so we have numbers only for the second half of the year.
The numbers are not so high as for the builds but still `18 498` test runs.

![Test runs in 2021](/images/2021-in-numbers-copr-builds.png)

## Top 20 projects in the number of PR Copr Builds

![Top20 projects in the number of PR Copr Builds](/images/2021-in-numbers-top20-copr-builds.png)

> ## Top 20 projects in the number of PR Test runs

Reviewer 2: ```suggestion
## Top 20 projects in the number of PR test runs
```

not the same as Copr which is an abbreviation
